docs/built_in_datasets.rst

```diff
@@ -13,8 +13,25 @@ to automate these operations.
 Environment variable
 --------------------
 
-In order for Fuel to know where to look for its data, the ``FUEL_DATA_PATH``
-environment variable has to be set.
+In order for Fuel to know where to look for its data, the ``data_path``
+configuration variable has to be set inside ``~/.fuelrc``. It's expected to be
+a sequence of paths separated by an OS-specific delimiter (``:`` for Linux and
+OSX, ``;`` for Windows):
+
+.. code-block:: yaml
+
+    # ~/.fuelrc
+    data_path: "/first/path/to/my/data:/second/path/to/my/data"
```

Comment:
Just occurred to me that it would make a lot of sense to accept a list directly here too (a lot nicer to edit, with the syntax I showed you below), but we can leave that for another PR.